src/nft/components/layout/Marquee.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box } from 'nft/components/Box'
+import * as styles from 'nft/components/layout/Marquee.css'
+import { ReactNode, useEffect, useRef, useState } from 'react'
+
+export const Marquee = ({ children, speed = 20 }: { children: ReactNode; speed?: number }) => {
+  const [duration, setDuration] = useState(0)
+  const containerRef = useRef<HTMLDivElement | null>(null)
+  const marqueeRef = useRef<HTMLDivElement | null>(null)
+
+  const updateScrollDuration = () => {
+    let containerWidth = 0
+    let marqueeWidth = 0
+
+    try {
+      if (marqueeRef.current && containerRef.current) {
+        containerWidth = containerRef.current.getBoundingClientRect().width
+        marqueeWidth = marqueeRef.current.getBoundingClientRect().width
+      }
+    } catch (e) {
+      console.error('Refs became unexpectedly null', e)
```

Comment:
I don't know that anything needs to be logged here. Can you remove the `console.error`?

Reply:
solved by setting duration to 0